integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24Tests.kt

```diff
@@ -320,36 +322,24 @@ private const val expectedSep24Info =
   {
     "deposit": {
       "JPYC": {
-        "enabled": true,
-        "min_amount": 1,
-        "max_amount": 1000000
```

Comment:
It would be good to keep at least one asset with `min_amount` and `max_amount` set. Same comment applies to other tests below.

Reply:
It's also possible for only one of `min_amount` or `max_amount` to be null, right? Can we include tests cases for that?

Reply:
Hey @philipliu, thanks for the comments
There are 3 assets configured in the project, and I removed the max and min configs from 2 of them and kept it on the third one.

About the test case, it's a good catch, I didn't think about the possibility. I'll work on that

Reply:
Good catch, it looks like with current version of code, the case when only one of min/max is set won't work

Reply:
Updated!
I didn't need to add new tests, because when min or max is missing it don't thrown NPE. But I configured the asset with only max or min to check. 👍🏻